vm/ubpf_jit_x86_64.c

```diff
@@ -491,12 +491,22 @@ translate(struct ubpf_vm *vm, struct jit_state *state, char **errmsg)
     emit1(state, 0xc3); /* ret */
 
     /* Division by zero handler */
+    const char div_by_zero_fmt[] = "uBPF error: division by zero at PC %u\n";
+    // Emit format string into machine code.
+    emit_bytes(state, (void*)div_by_zero_fmt, sizeof(div_by_zero_fmt));
```

Comment:
I was wondering if the LEA instruction and the data part (format string) could be stored separately. If the format string is emitted at a place that is guaranteed the instruction pointer will never go. If the address of that place can be computed at JIT time, the LEA instruction can read it like any other memory address?